lib/cc/pull_requests.rb

```diff
@@ -2,7 +2,28 @@ class CC::PullRequests < CC::Service
   def receive_test
     setup_http
 
-    receive_test_status
+    response = receive_test_status
```

Comment:
Ah, sorry, I confused myself, and was thinking of the new PR integration support for test coverage. You're quite right, ignore me.